src/js/controllers/imageCtrl.js

```diff
@@ -0,0 +1,28 @@
+(function (angular) {
+    angular
+        .module('flickrPOC')
+        .controller('imagesCtrl', imagesCtrl);
+
+    imagesCtrl.$inject = ['images', '$log'];
+
+    function imagesCtrl(images, $log) {
+        var vm = this;// jshint ignore:line
+
+        function activate(){
+
+            images.loadPics()
+                .then(function (res) {
+                    vm.images = res;
+                    return vm.images;
+                })
+                .catch(
+                    function(err){
+                        $log.error('Error loading images ', err.status, err.data);
```

Comment:
2 things:

1. Can you keep the callback on the same line as `.catch(`?
2. Can you add test(s) for the $log call?